src/language/fi.json

```diff
@@ -214,11 +214,11 @@
       "invalid-korvaus": "Käyttötarkoitusluokka ja postinumero eroavat, joten et voi korvata energiatodistusta, Tarkista korvattava energiatodistus."
     },
     "korvaava": {
-      "header": "Energiatodistuksen korvaava"
+      "header": "Korvaa energiatodistuksen"
     },
     "tila": {
       "draft": "Luonnos",
-      "in-signing": "Allekirjoitus",
+      "in-signing": "Allekirjoitus keskeytynyt",
```

Comment:
Onkohan tämä oikein?

Reply:
joo tila halutaan näyttää käyttäjälle näin

Reply:
itsekkään en hirveästi tuosta tykkää mutta onhan se totta että jos tässä tilassa tullaan et:lle niin tyypillisesti allekirjoitus on keskeytynyt

Reply:
tosin näytöllehän tulee nyt myös modaali dialogi, jossa tätä selvennetään